sw/cheri/sim_boot_stub/makefile

```diff
@@ -0,0 +1,9 @@
+CFLAGS=-target riscv32-unknown-unknown -mcpu=cheriot -mabi=cheriot \
```

Comment:
It'll be good that this uses cmake similar to other binaries that we have, but we can save that to a later PR.

Reply:
It uses make intentionally so you can build it in environments where you don't have cmake just to minimize dependencies required to build it